moveit_plugins/moveit_ros_control_interface/moveit_ros_control_interface_plugins.xml

```diff
@@ -4,4 +4,10 @@
         <description></description>
     </class>
   </library>
+
+  <library path="moveit_ros_control_interface_gripper_plugin">
+    <class name="position_controllers/GripperActionController" type="moveit_ros_control_interface::GripperControllerAllocator" base_class_type="moveit_ros_control_interface::ControllerHandleAllocator">
```

Comment:
You need to add another class for `effort_controllers/GripperActionController` see [ros_control_plugins.xml](https://github.com/ros-controls/ros2_controllers/blob/master/gripper_controllers/ros_control_plugins.xml)

Reply:
I will add that!